apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19482,5 +19479,44 @@
 <e>       <p><l>zigʻirkor<s n="adv"/></l>              <r>zıǵırkar<s n="adv"/></r></p></e>
 <e>       <p><l>zoʻraki<s n="adv"/></l>                <r>jasalma<s n="adv"/></r></p></e>
 
+<!-- GSoC 2020 -->
+<e>       <p><l>yana<s n="cnjcoo"/></l>                    <r>jáne<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>keyin<s n="adv"/></l>                   <r>keyin<s n="adv"/></r></p></e>
+<e>       <p><l>iliq<s n="adj"/></l>                    <r>jıllı<s n="adj"/></r></p></e>
+<e>       <p><l>ham<s n="cnjcoo"/></l>                  <r>hám<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>va<s n="cnjcoo"/></l>                   <r>hám<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>vaqt<s n="n"/></l>                      <r>waqıt<s n="n"/></r></p></e>
+<e>       <p><l>Asal<s n="np"/><s n="ant"/><s n="f"/></l>   <r>Asal<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>uy<s n="n"/></l>                        <r>úy<s n="n"/></r></p></e>
+<e>       <p><l>ol<s n="v"/><s n="tv"/></l>             <r>al<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>tashqari<s n="n"/></l>                  <r>sırt<s n="n"/></r></p></e>
+<e>       <p><l>Oygul<s n="np"/><s n="ant"/><s n="f"/></l> <r>Aygúl<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>Baxtiyor<s n="np"/><s n="ant"/><s n="m"/></l> <r>Baxtıyar<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>ular<s n="p3"/><s n="pl"/></l>       <r>olar<s n="p3"/><s n="pl"/></r></p></e>
+<e>       <p><l>ulkan<s n="adj"/></l>                    <r>úlken<s n="adj"/></r></p></e>
+<e>       <p><l>qachon<s n="adv"/><s n="itg"/></l>                   <r>qashan<s n="adv"/><s n="itg"/></r></p></e>
+<e>       <p><l>xursand<s n="adj"/></l>                    <r>quwanıshli<s n="adj"/></r></p></e>
+<e>       <p><l>ob-havo<s n="n"/></l>                  <r>hawa<b/>rayı<s n="n"/></r></p></e>
+<e>       <p><l>yaqinlash<s n="v"/><s n="iv"/></l>             <r>jaqınlas<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>bekit<s n="v"/><s n="tv"/></l>             <r>jasır<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>ber<s n="v"/><s n="tv"/></l>             <r>ber<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>yoki<s n="cnjcoo"/></l>                  <r>yamasa<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>yosh<s n="n"/></l>                  <r>jas<s n="n"/></r></p></e>
+<e>       <p><l>sen<s n="p2"/><s n="sg"/></l>       <r>sen<s n="p2"/><s n="sg"/></r></p></e>
+<e>       <p><l>yer<s n="n"/></l>                  <r>jer<s n="n"/></r></p></e>
+<e>       <p><l>kir<s n="v"/><s n="iv"/></l>             <r>kir<s n="v"/><s n="iv"/></r></p></e>
+<e>       <p><l>bolakay<s n="n"/></l>                     <r>bala<s n="n"/></r></p></e>
+<e>       <p><l>haliyam<s n="adv"/></l>                   <r>ele<s n="adv"/></r></p></e>
+<e>       <p><l>qanday<s n="adv"/></l>                   <r>qanday<s n="adv"/></r></p></e>
+<e>       <p><l>singli<s n="n"/></l>                     <r>qarındas<s n="n"/></r></p></e>
```

Comment:
singli should also translate to sińil, right?

Reply:
no.

Reply:
Isn't qarındas only a male's younger sister?  What about a female's younger sister?

Reply:
No, the meaning of sińli and qarındas  are the same. But there isn't a word "sińil" in kaa.

Reply:
Are sińli and qarındas really the same?  In Kazakh and Kyrgyz they both differ by whether they refer to the younger sister of a male (qarındas) or female (sińli).  Is this not the case in Qaraqalpaq?

Reply:
No, their meaning are the same. I couldn't find any refer to difference between them.

Reply:
Okay, then they should both be in the dictionary.